chapters/publication.tex

```diff
@@ -268,94 +267,91 @@ \section{Preparing a complete replication package}
+due to licensing agreements or ethical concerns.
 Even if you cannot release data immediately or publicly,
-there are often options to catalog or archive the data without open publication.
+there are often options to catalog or archive the data.
 These may take the form of metadata catalogs or embargoed releases.
 Such setups allow you to hold an archival version of your data
 which your publication can reference,
-as well as provide information about the contents of the datasets
+and provide information about the contents of the datasets
 and how future users might request permission to access them
-(even if you are not the person who can grant that permission).
+(even if you are not the person to grant that permission).
 They can also provide for timed future releases of datasets
 once the need for exclusive access has ended.
 
 If your project collected original data,
 releasing the cleaned dataset is a significant contribution that can be made
 in addition to any publication of analysis results.\sidenote{
   \url{https://www.povertyactionlab.org/sites/default/files/resources/J-PAL-guide-to-publishing-research-data.pdf}}
-Publishing data can foster collaboration with researchers
-interested in the same subjects as your team.
-Collaboration can enable your team to fully explore variables and
-questions that you may not have time to focus on otherwise,
-even though data was collected on them.
+It allows other researchers to validate the mechanical construction of your results,
+to investigate what other results might be obtained from the same population,
+and test alternative approaches or answer other questions.
+This fosters collaboration and may enable your team to fully explore variables and
+questions that you may not have time to focus on otherwise.
 There are different options for data publication.
 The World Bank's Development Data Hub\sidenote{
-	\url{https://data.worldbank.org}}
+	\url{https://datacatalog.worldbank.org}}
 includes a Microdata Catalog\sidenote{
 \url{https://microdata.worldbank.org}}
+and a Geospatial Catalog,
 where researchers can publish data and documentation for their projects.\sidenote{
 \url{https://dimewiki.worldbank.org/Microdata\_Catalog}
 \newline
 \url{https://dimewiki.worldbank.org/Checklist:\_Microdata\_Catalog\_submission}
 }
 The Harvard Dataverse\sidenote{
 	\url{https://dataverse.harvard.edu}}
-publishes both data and code,
-and also creates a data citation for its entries --
-IPA/J-PAL field experiment repository is especially relevant\sidenote{
-	\url{https://www.povertyactionlab.org/blog/9-11-19/new-hub-data-randomized-evaluations}}
-for those interested in impact evaluation.
+publishes both data and code. 
+The Datahub for Field Experiments in Economics and Public Policy\sidenote{\url{https://dataverse.harvard.edu/dataverse/DFEEP}} 
+is especially relevant for impact evaluations. 
+Both the World Bank Microdata Catalog and the Harvard Dataverse
+create data citations for deposited entries.
```

Comment:
Add line here to resolve #371 :

DIME has its own collection of data sets in the Microdata Catalog,
where data from our projects is published.\sidenote{\url{
https://microdata.worldbank.org/catalog/dime}}

Reply:
done in 90a7e953f8ab0a71754e1ad75188bee681ab764f